### PR TITLE
Docs: update documentation for Icons 

### PR DESCRIPTION
**Pull request by monicawheeler**
## Description
Adjust the location of icons as referenced throughout the documentation

## Testing in `sage-lib`
Copy changes only

## Testing in `kajabi-products`
No external impact. Doc changes only.

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]

**Comment by ju-Skinner** [comment]
Is this to adjust the text used? However, if it's supposed to reference a new location, then I have an issue because  I don't see anything pointing to the new Icon documentation in Pine.

File: docs/app/views/examples/components/icon/_props.html.erb [hunk 1]
```diff
@@ -30,14 +30,14 @@
 </tr>
 <tr>
   <td><%= md('`color`') %></td>
-  <td><%= md('Which color to use to render the icon. See Sage Colors under "Design."') %></td>
+  <td><%= md('Which color to use to render the icon. See Sage Colors under "Foundations."') %></td>
```
Comment (ju-Skinner) [inline docs/app/views/examples/components/icon/_props.html.erb:33]: I'm glad to see you added this in. It was a request in Sage Support yesterday.

**Comment by monicawheeler** [comment]
> Is this to adjust the text used? However, if it's supposed to reference a new location, then I have an issue because I don't see anything pointing to the new Icon documentation in Pine.

The updates here were meant to adjust the language to reference “Icons” in the Foundations page. I could clarify that in all areas by saying “See Foundations / Icons”. I also see the need to add information directing users to Pine as well. What do you think?

**Comment by ju-Skinner** [comment]
It was the title of the PR that threw me off.

**Pull request merged**